Jellyfin.Api/Helpers/RequestHelpers.cs

```diff
@@ -25,5 +27,18 @@ internal static string[] Split(string value, char separator, bool removeEmpty)
                 ? value.Split(new[] { separator }, StringSplitOptions.RemoveEmptyEntries)
                 : value.Split(separator);
         }
+
+        internal static SessionInfo GetSession(ISessionContext sessionContext)
+        {
+            // TODO: how do we get a SessionInfo without IRequest?
+            SessionInfo session = sessionContext.GetSession("Request");
```

Comment:
The new infrastructure has a different `Request `object which I don't think is compatible with `GetSession`